src/test/java/org/openrewrite/staticanalysis/LambdaBlockToExpressionTest.java

```diff
@@ -25,11 +26,12 @@
 import static org.openrewrite.java.Assertions.java;
 
 class LambdaBlockToExpressionTest implements RewriteTest {
-
     @Override
     public void defaults(RecipeSpec spec) {
         spec.recipe(new LambdaBlockToExpression())
-            .parser(JavaParser.fromJavaVersion().logCompilationWarningsAndErrors(true));
+            .parser(JavaParser.fromJavaVersion()
+              .classpathFromResources(new InMemoryExecutionContext(), "junit-jupiter-api-5.10")
```

Comment:
Since this dependency is only used for tests, we should make sure to not add it to `src/main/resources`.